perf/ring.go

```diff
@@ -79,7 +79,7 @@ func perfBufferSize(perCPUBuffer int) int {
 func (ring *perfEventRing) Close() {
 	runtime.SetFinalizer(ring, nil)
 	unix.Close(ring.fd)
-	unix.Munmap(ring.mmap)
+	_ = unix.Munmap(ring.mmap)
```

Comment:
I can't find the reason for `Close()` being ignored by errcheck in either the errcheck or the golangci-lint source, will try to find out why that is. There are a couple of threads complaining about errcheck forcing deferred Close()s to be error-checked as well, but this is not the same.

So yes, the sole reason why I didn't add a check on Close() as well is because the linter seems to be okay with it. Should we make `(perfEventRing).Close()` return error, or is it really fine if Close and Munmap return errors?

Reply:
Opted not to break `ring.Close()` API, if this proves an issue later, we could return error.